dockstore-webservice/src/main/java/io/dockstore/webservice/resources/OrganizationResource.java

```diff
@@ -409,6 +410,37 @@ private Organization getOrganizationByIdOptionalAuth(Optional<User> user, Long o
         }
     }
 
+    @DELETE
+    @Path("/{organizationId}")
+    @Timed
+    @UnitOfWork
+    @ApiOperation(value = "Deletes an organization that is pending or rejected", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, hidden = true)
```

Comment:
I think this annotation may be confusing (people may be under the impression that adding/removing anything here will have some effect.  Just make it `@ApiOperation(value = "hidden", hidden = true)`

Reply:
Removed the authorization of `@ApiOperation` and simplified it to just display hidden for its value